x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/components/agent_policy_advanced_fields/hooks.tsx

```diff
@@ -140,6 +140,12 @@ export function useOutputOptions(agentPolicy: Partial<NewAgentPolicy | AgentPoli
           value: item.id,
           inputDisplay: item.name,
           disabled: !isPolicyPerOutputAllowed || isInternalOutput,
+          helpText: isPolicyPerOutputAllowed ? undefined : (
```

Comment:
`helpText` is not supported by the `EuiSuperSelect` where this config is used.
I could get it working locally by changing the `inputDisplay` (should be used at 2 paces where `isPolicyPerOutputAllowed` is used).
And I think the license check should be used as a separate condition, because `isPolicyPerOutputAllowed` can be true due to other reasons too.
 `let hasMinimumLicense = licenseService.hasAtLeast(LICENCE_FOR_PER_POLICY_OUTPUT);`

```
inputDisplay: getOutputLabel(
            item.name,
            !hasMinimumLicense ?  <FormattedMessage
            id="xpack.fleet.agentPolicyForm.outputOptionDisabledPolicyPerOutput"
            defaultMessage="The current license does not allow defining output per policy."
          /> 
          : undefined
          )
```

<img width="904" alt="image" src="https://github.com/elastic/kibana/assets/90178898/096a2470-300b-4535-9b0f-724e90b8887b">

Reply:
Thanks @juliaElastic, I've pushed a commit with your changes.